src/pages/Sketch/components/Canvas.jsx

```diff
@@ -461,14 +464,16 @@ function Canvas() {
       accepts: [
         {
           description: "Doc2pen Sketch Save File",
-          extensions: ["d2ps"],
-          mimeTypes: ["application/d2ps"],
+          extensions: [".png"],
+          mimeTypes: ["image/*"],
```

Comment:
This one too

Reply:
Moreover, `getNewFileHandle` is meant to **SAVE** d2ps file. It is not meant to **LOAD** image files.

So a completely irrelevant function is edited. The changes in this function are Obsolete